src/map.rs

```diff
@@ -3281,13 +3281,13 @@ mod tests {
 /// drop(r);
 /// ```
 ///
-/// # Keys and values must be static
+/// # Keys and values do not have be static
 ///
-/// ```compile_fail
+/// ```no_run
```

Comment:
Why make these `no_run`? Can't they just be regular doctests?

Reply:
True, I was just following the "get() can be non 'static test below".

Reply:
Updated in 5db3eb82c3707fab6c788cd2d802f1e7288c2ce3.